sql-parser/src/test/java/io/crate/sql/parser/TestStatementBuilder.java

```diff
@@ -717,6 +717,16 @@ public void testInsertStmtBuilder() throws Exception {
         printStatement("insert into t (a, b) values (1, 2), (3, 4) on duplicate key update a = values (a) + 1, b = 4");
         printStatement("insert into t (a, b) values (1, 2), (3, 4) on duplicate key update a = values (a) + 1, b = values(b) - 2");
 
+        try {
+            printStatement("insert into t (a, b) values (1, 2) on conflict do nothing");
+        } catch (UnsupportedOperationException e) {
+            // this is what we want
+        }
+        printStatement("insert into t (a, b) values (1, 2) on conflict do update set a = a + 1");
+        printStatement("insert into t (a, b) values (1, 2) on conflict do update set a = a + 1, b = 3");
+        printStatement("insert into t (a, b) values (1, 2), (3, 4) on conflict do update set a = values (a) + 1, b = 4");
+        printStatement("insert into t (a, b) values (1, 2), (3, 4) on conflict do update set a = values (a) + 1, b = values(b) - 2");
```

Comment:
the `on conflict` syntax doesn't use `values(column)` but instead supports a `excluded` relation. To be compatible we'll need to support that (and ideally disallow the `values(col)` syntax if used as part of the `on conflict`)

Reply:
https://www.postgresql.org/docs/9.5/static/sql-insert.html

I see it using VALUES?

Reply:
Ah I see, you mean inside the on conflict. Let me fix that.

Reply:
I've modified the test to use excluded. The analyzer will perform the values check which is out of scope for this PR.